gke/README.md

```diff
@@ -51,8 +51,36 @@ The following describes the setup process for this project:
 1. Install Jenkins on GKE using the stable/jenkins helm chart: [Jenkins On GKE Tutorial](
    https://cloud.google.com/solutions/jenkins-on-kubernetes-engine-tutorial).
 
-1. Install and configure the GKE plugin: [Jenkins GKE Docs](
-   https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+1. Install and configure the following plugins:
+   1. GKE plugin: [Jenkins GKE Docs](https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+   1. GCS plugin: [Jenkins GCS Docs](https://github.com/jenkinsci/google-storage-plugin/blob/develop/README.md).
+   1. Global Variable String Parameter plugin: [Jenkins Global Var Docs](https://wiki.jenkins.io/display/JENKINS/Global+Variable+String+Parameter+Plugin).
```

Comment:
The global var param plugin isn't used in any of the downstream steps. Adding it as a requirement seems unnecessary.

Reply:
I believe you need it for step 6, adding global parameters.  Can you check your installation?  If you don't have, let's remove this step.

Reply:
My installation does not have it.